forge-gui/res/cardsfolder/upcoming/the_war_games.txt

```diff
@@ -0,0 +1,13 @@
+Name:The War Games
+ManaCost:2 W W
+Types:Enchantment Saga
+K:Saga:4:DBTokens,DBPutCounterAll,DBPutCounterAll,DBEffect
+SVar:DBTokens:DB$ Token | TokenAmount$ 3 | TokenScript$ w_1_1_warrior | TokenOwner$ Player | RememberTokens$ True | SubAbility$ DBGoad | SpellDescription$ Each player creates three tapped 1/1 white Warrior creature tokens. The tokens are goaded for as long as CARDNAME remains on the battlefield.
+SVar:DBGoad:DB$ Goad | Defined$ Remembered | Duration$ UntilHostLeavesPlay | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBPutCounterAll:DB$ PutCounterAll | ValidCards$ Creature.Warrior | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on each Warrior creature.
+SVar:DBEffect:DB$ Effect | Execute$ TrigImmediate | SpellDescription$ You may exile a nontoken creature you control. When you do, exile all Warriors.
```

Comment:
don't need this noop effect

Reply:
What is noop?

Reply:
it's computer science jargon for no operation
= eliminate this line

Reply:
What line, precisely, should I remove? Should I just have the saga hook directly into the line with the cost? And in that case, should I change it from AB to DB?

Reply:
the only thing DBEffect provides is the SpellDescription, which could just be moved onto the following line
in fact, having a cost in the middle of an effect chain this way shouldn't even work

Reply:
SubAbility = chain stuff together
Execute = tell a trigger what it's resolving effect is